hoptimator-util/src/main/java/com/linkedin/hoptimator/util/planner/ScriptImplementor.java

```diff
@@ -166,11 +166,13 @@ public QueryImplementor(RelNode relNode) {
     public void implement(SqlWriter w) {
       RelToSqlConverter converter = new RelToSqlConverter(w.getDialect());
       SqlImplementor.Result result = converter.visitRoot(relNode);
-      SqlSelect select = result.asSelect();
-      if (select.getSelectList() != null) {
+      SqlNode node = result.asStatement();
+      if (node instanceof SqlSelect && ((SqlSelect) node).getSelectList() != null) {
+        SqlSelect select = (SqlSelect) node;
         select.setSelectList((SqlNodeList) select.getSelectList().accept(REMOVE_ROW_CONSTRUCTOR));
       }
-      select.accept(new UnflattenMemberAccess(this)).unparse(w, 0, 0);
+//     node.accept(new UnflattenMemberAccess(this)).unparse(w, 0, 0);
```

Comment:
ditto

Reply:
Do we need `UnflattenMemberAccess` now? I suspect there is still stuff to touch up in the output SQL?

Reply:
Hmm I don't think there is anymore, at least once I push the change on top of this to fix up the venice key stuff

Reply:
cool, lemme clean this up and push again